src/UI/Components/AvatarCircle/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Image, ImageSourcePropType, View } from 'react-native';
+import Svg, { Circle as SvgCircle } from 'react-native-svg';
+import { styles } from './styles';
+
+interface CircleProps {
+    radius: number; // The radius of the circle
+    imageSource: ImageSourcePropType; // The image source, which can be a local image or a URL
+  }
+
+/**
+ * Circle component that displays an image within a circular frame.
+ * 
+ * @param {CircleProps} props - Props for the Circle component.
+ * @returns {JSX.Element} A circle component with an image inside.
+ */
+const AvatarCircle: React.FC<CircleProps> = ({ radius, imageSource }) => {
```

Comment:
There is a small warning for doc: JSDoc types may be moved to TypeScript types. Do handle this.